Team417_CENTERSTAGE/src/main/java/org/firstinspires/ftc/team417_CENTERSTAGE/pidf/PIDFArmController.java

```diff
@@ -0,0 +1,20 @@
+package org.firstinspires.ftc.team417_CENTERSTAGE.pidf;
+
+import com.qualcomm.robotcore.eventloop.opmode.Disabled;
+import com.qualcomm.robotcore.eventloop.opmode.TeleOp;
+
+import org.firstinspires.ftc.team417_CENTERSTAGE.baseprograms.BaseOpMode;
+
+@Disabled
```

Comment:
I think this is a test opmode (it needs a comment!) in which case you could consider putting it in the "testing" portion/package.

Reply:
👍